File: js/linked-list/linked-list-operations.js

class NodeOpt {
    constructor(data) {
        this.data = data;
        this.next = null;
    }
}

node1 = new NodeOpt(7);
node2 = new NodeOpt(11);
node3 = new NodeOpt(3);
node4 = new NodeOpt(2);
node5 = new NodeOpt(9);

node1.next = node2;
node2.next = node3;
node3.next = node4;
node4.next = node5;

File: python/linked-lists/linked-list-operations.py
class NodeOpt:
    def __init__(self, data):
        self.prev = None
        self.data = data
        self.next = None

node1 = NodeOpt(3)
node2 = NodeOpt(5)
node3 = NodeOpt(13)
node4 = NodeOpt(2)

node1.next = node2
node2.next = node3
node3.next = node4

